gstd/Cargo.toml

```diff
@@ -21,6 +21,10 @@ futures = { workspace = true, features = ["alloc"] }
 static_assertions.workspace = true
 
 [features]
-default = ["panic-handler"]
+default = ["default-stable", "nightly"]
+default-stable = ["panic-handler"]
+nightly = ["panic-messages", "oom-handler"]
```

Comment:
What about this?
```suggestion
default = ["stable", "nightly"]
stable = ["panic-handler"]
nightly = ["panic-messages", "oom-handler"]
```

Reply:
The current name highlights the fact that `default-stable` is actually a default set of features that compiles on stable; `stable` wouldn't be as apparent

Reply:
> The current name highlights the fact that `default-stable` is actually a default set of features that compiles on stable; `stable` wouldn't be as apparent

why "nightly" is intuitive than?

Reply:
`nightly` is a feature that enables all nightly features, `default-nightly` would be "default features for nightly toolchain" - that's covered by `default`. In some sense, `nightly` is a "modifier" unlike `default-stable` which is a basic feature set.